### PR TITLE
Made basic changes to README

### PR DESCRIPTION
**Pull request by sameetandpotatoes**
- Minor deployment README changes
- Probably should get these out before the next meeting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,28 +26,27 @@ Questions on how to add your app to Groot or use the Groot API:
     sudo apt install repo
 
     ```
-2. Make a directory to house your groot work
+2. Clone this repository
```
Comment (narendasan) [inline README.md]: You should not need to clone the repo, the repo sync should pull down deploy as well now. Just make a directory, init and sync

File: README.md [hunk 2]
```diff
@@ -56,7 +55,7 @@ Questions on how to add your app to Groot or use the Groot API:
     && mkdir [SOME DIRECTORY]/src
     mkdir -p [SOME DIRECTORY]/src/github.com/acm-uiuc
     ```
```
Comment (narendasan) [inline README.md]: I tend to keep path stuff in my .profile would at least keep the references

File: README.md [hunk 3]
```diff
@@ -122,19 +121,19 @@ Questions on how to add your app to Groot or use the Groot API:
     source ~/.profile
     nvm install 7.2.0
     nvm use 7.2.0
-    sudo npm install -g forever
```
Comment (narendasan) [inline README.md]: Do you need this for just working on the project (btw will probably rename this repo to groot after the arbor changes are done)

File: README.md [hunk 4]
```diff
@@ -145,31 +144,29 @@ Questions on how to add your app to Groot or use the Groot API:
...
     ```
     
 - MySQL
     + Install Software
     ```sh
-    #macOS
+    # macOS
     brew install mysql
-    mysqladmin -u root password 'yourpassword' 
+    mysqladmin -u root password ''
```
Comment (narendasan) [inline README.md]: Having a password should be recommended

**Comment by sameetandpotatoes** [comment]
Yeah, a lot of these changes are a bit deprecated since Ben's PR covers most of these. After his is merged, I'll rebase and fix anything resulting after that.

**Comment by sameetandpotatoes** [comment]
@narendasan Updated based on Ben's merge - moving first time setup out of the main installation, as we want to encourage setup with Docker.

**Comment by narendasan** [comment]
regardless just in case, make sure the instructions for the other way are correct (just perhaps make a docker install (recommended) and regular install section)

**Comment by sameetandpotatoes** [comment]
Yep, that's exactly what I went with. At the first meeting, we did use the regular installation instructions and they did work. It just gets tedious very quickly setting up each service

File: README.md [hunk 5]
```diff
@@ -25,150 +25,21 @@ Questions on how to add your app to Groot or use the Groot API:
     ```sh    
     sudo apt install repo
 
-    ```
-2. Make a directory to house your groot work
```
Comment (narendasan) [inline README.md]: section is still required

File: README.md [hunk 6]
```diff
@@ -25,150 +25,21 @@ Questions on how to add your app to Groot or use the Groot API:
...
-    ```sh
-    mkdir groot
-    ```
-    
-3. Within this directory run the following command to start managing the projects
+    ```    
+2. Within this directory run the following command to start managing the projects
 
     ```sh    
+    cd groot-deploy
```
Comment (narendasan) [inline README.md]: change to cd groot

**Comment by sameetandpotatoes** [comment]
Gonna close this, kind of outdated now

**Pull request closed without merging**